less/matching.less

```diff
@@ -21,8 +21,7 @@
 		margin-top: -(@icon-size/2);
 		right: 10px;
 		background: none;
-		z-index: 1;
-
+		
```

Comment:
Bigger fish to fry and all that but can we lose the evil leading whitespace that's been added here?